packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -93,17 +97,12 @@ describe('Ceramic integration', () => {
     }
 
     const findPort = async (start: number, offset: number): Promise<number> => {
-        return await getPort({port: getPort.makeRange(start, start + offset)})
+      return await getPort({port: getPort.makeRange(start + 1, start + offset)})
     }
 
     ([port1, port2, port3] = await Promise.all([p1Start, p2Start, p3Start].map(start => findPort(start, pOffset))));
     ([ipfs1, ipfs2, ipfs3] = await Promise.all([port1, port2, port3].map(port => createIPFS(buildConfig(tmpFolder.path, port)))));
-    ([p1Start, p2Start, p3Start] = [p1Start, p2Start, p3Start].map(start => start + 1))
-
-    // Wait for all IPFS nodes to be fully started up.
-    await ipfs1.config.getAll();
```

Comment:
What happens if you replace these calls to getAll() with a single call to delay(1000), and remove the delay calls from the test bodies?  Would be nice to handle this in the generic beforeEach part if possible so it doesn't have to be remember to be done in each individual test.